### ISSUE TITLE
Label Smoothing Regularization

### ISSUE THREAD

**Issue by lorinczszabolcs** [open-mmlab/mmsegmentation#1657]
# Support label smoothing regularization

**Motivation**
Label smoothing has been shown to boost performance in Inception-V2, as well as in Revisiting ResNets papers.

**Related resources**
PyTorch Cross-Entropy Loss already supports it, by passing label_smoothing parameter between 0.0 and 1.0. The Cross-Entropy implementation in mmsegmentation could take additional kwargs and pass them forward to the pytorch functions.

**Comment by MengzhangLI** [comment]
Hi, it is a good suggestion. We would consider to add this feature in cross entropy loss.
